app/src/main/java/com/muzima/view/forms/HTMLFormDataStore.java

```diff
@@ -1248,13 +1248,58 @@ public String getDerivedObservationByPatientUuidAndDerivedConceptId(String patie
         return createDerivedObsJsonArray(derivedObservations);
     }
 
+    @JavascriptInterface
+    public String getInterventionsDerivedObservationByPatientUuid(String patientUuid) throws JSONException, DerivedConceptController.DerivedConceptFetchException {
+        List<DerivedObservation> derivedObservations = new ArrayList<>();
+        try {
+            derivedObservations = derivedObservationController.getDerivedObservationByPatientUuid(patientUuid);
+            derivedObservations.addAll(derivedObservationController.getDerivedObservationByPatientUuidAndDerivedConceptUuid(patientUuid, "4b479a6c-4276-45a1-b785-ecbc7dc59ff1"));
```

Comment:
If possible check if the result is empty before doing addAll for all the addAll calls